addons/weave-net/weave-net.yaml

```diff
@@ -92,14 +100,18 @@ items:
       - kind: ServiceAccount
         name: weave-net
         namespace: kube-system
-  - apiVersion: extensions/v1beta1
+  - apiVersion: apps/v1
     kind: DaemonSet
     metadata:
       name: weave-net
       labels:
         name: weave-net
       namespace: kube-system
     spec:
+      # Wait 5 seconds to let pod connect before rolling next pod
```

Comment:
This comment is misplaced

Reply:
Copy-n-paste from here:
https://github.com/weaveworks/weave/blob/master/prog/weave-kube/weave-daemonset-k8s-1.11.yaml#L111

I'd like to move it down to the correct place, but I think it's better to be the same as the source, so that we can diff in the future.